client/css/style.css

```diff
@@ -1643,11 +1643,16 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	opacity: 1;
 }
 
-.textcomplete-item .emoji {
-	margin-right: 8px;
-	width: 16px;
-	text-align: center;
+.inline-emoji {
 	display: inline-block;
+	font-size: 1.4em;
+	vertical-align: bottom;
+	line-height: 1;
+}
+
+.textcomplete-item .inline-emoji {
```

Comment:
As far as I can tell, before your PR, emoji and text were both vertically misaligned in dropdown, and after your PR, emoji is nicely centered, but not text. Because they're not aligned with each other anymore, it jumped on me, but never realized both weren't correctly aligned before.
Any chance you could attempt to vertically center the text too?

Before | After
--- | ---
<img width="196" alt="screen shot 2017-08-23 at 23 40 28" src="https://user-images.githubusercontent.com/113730/29648576-fc60af02-885c-11e7-825c-4a4472ee80cd.png"> | <img width="205" alt="screen shot 2017-08-23 at 23 40 47" src="https://user-images.githubusercontent.com/113730/29648577-fc6373e0-885c-11e7-8d48-26e0e855a758.png">

Reply:
@xPaw, did you manage to fix this?

Reply:
I think so.